runelite-client/src/main/java/net/runelite/client/plugins/boosts/BoostsConfig.java

```diff
@@ -64,4 +65,70 @@ default boolean displayIndicators()
 	{
 		return false;
 	}
+
+	@ConfigItem(
```

Comment:
I dont think coloring hihg/med/low boosts with having it configurable is good idea. People are used to green if good, red if bad, thats all.